components/shortcuts_modal/shortcuts_modal.jsx

```diff
@@ -303,9 +276,8 @@ class ShortcutsModal extends React.PureComponent {
         return (
             <Modal
                 dialogClassName='a11y__modal shortcuts-modal'
-                show={this.state.show}
-                onHide={this.handleHide}
-                onExited={this.handleHide}
```

Comment:
I think we still need to have this `onExited` callback. Was this an intended change?

Reply:
I think the lifecycle is onHide is called, then onExited. Previously both these functions just set local modal state of `show` to `false`. Is there a case when `onExited` is called but not `onHide`? Maybe only `onExited` is needed. Happy to add back.